test/jest_integration/saml_multiauth.test.ts

```diff
@@ -274,6 +275,15 @@ describe('start OpenSearch Dashboards server', () => {
       until.elementsLocated(By.xpath('/html/body/div[1]/div/header/div/div[2]')),
       20000
     );
+    await driver.wait(until.elementsLocated(By.css('img[data-test-subj="defaultLogo"]')), 20000);
+    await driver.wait(
+      until.elementsLocated(By.css('section[aria-labelledby="homDataAdd__title"]')),
```

Comment:
nit: _home_

Reply:
Same below too

Reply:
Ah I think I couldn't rename this since the element was defined with this name: 
![Screenshot 2023-11-14 at 9 40 40 AM](https://github.com/opensearch-project/security-dashboards-plugin/assets/109499885/9d65b6a4-7a2b-43e6-917e-0249a4c69d86)